src/mapscript/swiginc/image.i

```diff
@@ -250,14 +250,14 @@
             msSetError(MS_IMGERR, "Failed writing image to %s",
                        "imageObj::saveWebImage",
                        imageFile);
+            msFree(imageFile);                       
```

Comment:
apologies, the output of msBuildPath() shouldn't be msFree()'d after all (doh, let's just rewrite MapServer in Rust to avoid all those stupid errors)

```suggestion
```